.github/workflows/pr-test.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - 'v*'
+  merge_group:
```

Comment:
saw this as mentioned in the DCO issue, the recommendation from GH is to use `merge_group: ["main"]` :)